### ISSUE TITLE
failed to download crates

### ISSUE THREAD

**Issue by Tristan1900** [xd009642/tarpaulin#363]
Hi there, I having been following the [blog post](https://eipi.xyz/blog/rust-code-coverage-with-github-workflows/) and trying this nice tool on github action but found the following error
```
[INFO tarpaulin] Creating config
[INFO tarpaulin] Running Tarpaulin
[INFO tarpaulin] Building project
[INFO tarpaulin] Failure Failed to compile tests! Error: failed to download from `https://crates.io/api/v1/crates/futures/0.3.4/download`
Error: "Failed to report coverage! Error: No coverage results collected."
```
Sorry I couldn't paste the link since it's a private repo.

Notice that it's not specific to this one crate, everytime I run the crate will change. And I'm able to run tarpaulin successfully in my local environment. Any idea what might go wrong on github action?
Thanks!

**Comment by xd009642** [comment]
Could it be down to this issue? https://github.com/xd009642/tarpaulin/issues/319#issuecomment-593937845 if so when the next cargo release comes out it should be fixed. You can always fork tarpaulin install cargo from the git source and see if that fixes it. I'd try it myself but as you say it's a private repo so I'm not sure what the exact environment etc is

**Comment by jmagnuson** [comment]
I've been running into download timeouts in CI as well (though not for local runs). I believe it is related to https://github.com/rust-lang/cargo/issues/7974. Try running with `export CARGO_HTTP_MULTIPLEXING=false` to see if the issue persists.

**Comment by xd009642** [comment]
This should be fixed in the 0.11.1 release, so I'm going to close this issue. If it's not sorted feel free to reopen and comment :+1:
